packages/legacy/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@rollup/plugin-legacy",
+  "version": "1.0.0",
```

Comment:
Prior to migration this plugin used `rollup@^0.36.3` for testing and had no peerDeps. So that is technically a breaking change. Since we're deprecating the old module and asking folks to move to the new scoped module, we want to make that as frictionless as possible. Bumping the major version and adding a `Breaking Change` to the Change log is a good practice.

_Note that we just started doing this, was something that we missed in the beginning_